circle.yml

```diff
@@ -0,0 +1,19 @@
+machine:
+  services:
+    - docker
+
+test:
```

Comment:
We found a simple example of testing via `curl` here: https://circleci.com/docs/docker/#deployment-to-a-docker-registry